src/serde/de_br.rs

```diff
@@ -19,6 +20,7 @@ enum ParseOp {
 pub fn node_from_stream_backrefs(
     allocator: &mut Allocator,
     f: &mut Cursor<&[u8]>,
+    mut visitor: impl FnMut(NodePtr),
```

Comment:
I'm a bit puzzled by this name... would it be more accurate to call it `visit_callback` or something?

Reply:
`visited_accumulator`?

Reply:
`backref_callback`?